### PR TITLE
[BOUNTY] Holy Light sect, dedicated to healing

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

This is a privately paid bounty, council has been pinged about it

As per https://forums.yogstation.net/threads/vote-require-new-features-added-via-bounties-to-have-accompaning-documentation-to-be-paid-out.19311/ I will add this to the wiki if it is merged

Adds a new sect, Holy Light, dedicated to healing

Heals twice as much as normal, but imposes a 12 second cooldown

Heals all limbs, not just organic or robotic ones

Has a 20% chance to inflict brain damage instead of 40%

Favor is earned by healing. For each point of damage healed with the bible, you earn favor.

### Angelic Assistance

- Creates a medibot called "Holy Medibot"
- Costs 150 favor
- Takes 15 seconds

### Holy Sight

- Creates a one-use medhud implant autosurgeon
- Costs 250 favor
- Takes 15 seconds

### The White

- Creates a Rod of the White which acts as a rod of asclepius but cooler and heals at half the rate
- Costs 400 favor
- Takes 20 seconds

inactive

active

### Rebirth

- Fully heals a buckled mob. If they were dead, also inflicts 75 clone damage. Also applies a white color to them. Healed mob is told that they "forget all that had previously harmed them".
- Costs 650 favor
- Takes 60 seconds

# Spriting

# Changelog

:cl:  
rscadd: Added the Holy Light sect
/:cl:

### CODE REVIEW

**Comment by Hislyna** [comment]
Absolute legend

**Comment by ToasterBiome** [comment]
cool, rebirth effect might be a bit cringe

**Comment by Hislyna** [comment]
> cool, rebirth effect might be a bit cringe

Im gonna make sure you get that effect every round

**Comment by Moltijoe** [comment]
this might actually make me play chaplain

File: code/modules/religion/religion_sects.dm [hunk 1]
```diff
@@ -424,3 +424,45 @@
...
+	desc = "A sect dedicated to healing."
+	convert_opener = "Welcome to the Holy Light, disciple. Heal others to gain favor."
+	alignment = ALIGNMENT_GOOD // literally the only good sect besides default lol
+	rites_list = list(/datum/religion_rites/medibot, /datum/religion_rites/holysight, /datum/religion_rites/healrod, /datum/religion_rites/holyrevival)
+	altar_icon_state = "convertaltar-heal"
+	COOLDOWN_DECLARE(last_heal)
+
+/datum/religion_sect/holylight/on_conversion(mob/living/L)
+	. = ..()
+	for(var/obj/item/storage/book/bible/da_bible in L.get_contents())
```
Comment (Moltijoe) [inline code/modules/religion/religion_sects.dm]:
```suggestion
	for(var/obj/item/storage/book/bible/da_bible in L.GetAllContents())
```

Comment (Moltijoe) [reply code/modules/religion/religion_sects.dm]: in case they shove it in a box inside their backpack or something

**Review by Moltijoe — CHANGES_REQUESTED** [review]

**Review by cuackles — APPROVED** [review]
few notes but nothing negative, first thing is that you gave the altar an unique logo icon thingy sing whateverthefuck its called, the cross thingy, so like, good cus 3 of the other religions use the same but diferent color, the staff very good it has the same desing of the other so u can easely tell at a glance that it should do the same or similar stuff, and the white overlay on poeple also looks good i worry that maybe on some characters it will look odd but that can be revised or changed if thats the case and its nothing really to worry about, so anyway yeah good stuff

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**